metadata-ws/src/main/java/uk/ac/ebi/ampt2d/metadata/persistence/services/StudyServiceImpl.java

```diff
@@ -93,20 +96,22 @@ public int compare(Study o1, Study o2) {
 
     @Override
     public List<Study> findStudiesByTaxonomyId(long id) {
-        QStudy study = QStudy.study;
-        Predicate predicate = study.taxonomy.taxonomyId.eq(id).
-                or(study.taxonomy.ancestors.any().taxonomyId.eq(id));
-
-        return findStudiesByPredicate(predicate);
+        List<Long> speciesAndSubspeciesTaxonomyIds =
+                taxonomyTreeService.findAllSpeciesAndSubspeciesTaxonomyIdsInATaxonomyTreeByTaxonomyId(id);
```

Comment:
There are still references to "species and subspecies" throughout the code, even though we decided not to handle species/subspecies in any special way; we're just searching by a taxonomy ID